rosidl_typesupport_connext_cpp/rosidl_typesupport_connext_cpp/__init__.py

```diff
@@ -110,11 +110,72 @@ def generate_dds_connext_cpp(
             # modify generated code to avoid unsed global variable warning
             # which can't be suppressed non-globally with gcc
             msg_filename = os.path.join(output_path, msg_name + '.h')
+            # TODO(karsten1987): Modify should take array of callbacks
+            # to avoid multiple file readings
             _modify(msg_filename, pkg_name, msg_name, _inject_unused_attribute)
 
+            plugin_filename = os.path.join(output_path, msg_name + 'Plugin.cxx')
+            _modify(plugin_filename, pkg_name, msg_name, _modify_plugin_serialize_function)
+
     return 0
 
 
+def _get_serialization_code(msg_name, indentation):
+    val = ("{{\n"
```

Comment:
In order to avoid an indentation which is dependent on the variable name `val` add a newline after the open parenthesis.